### ISSUE TITLE
pagamento com dois cartões com pagseguro, cielo e mercado pago

### ISSUE THREAD

**Issue by thadeu220** [claudiosanches/woocommerce-pagseguro#118]
seria muito bacana que, nas lojas woocommerce, fosse possível inserir um sistema de duplo cartão de crédito, porém gratuito.
A ideia é permitir que, o cliente pague com dois cartões, podendo escolher quanto de cada cartão o sistema deve cobrar de cada cartão, até concluir o valor cheio...
Eu vejo grandes empresas fazerem isso, até o mercadolivre permite dois cartões, seria muito interessante que pudessemos fazer isso com o woocommerce, e espero que o Cláudio sanchez possa contribuir com a comunidade.
O Brazillian Market têm foco em atender as tendências do mercado brasileiro, e dois cartões é uma tendência.
Então, seria interessante que os módulos woocommerce projetados pelo Cláudio, na sua versão transparente, na medida do possível, permitissem dois cartões.

Espero notícias positivas.
se a área issues não for a área certa pra sugestões, me indiquem a área certa...
Agradeço!

**Comment by lwxbr** [comment]
@thadeu220. Acredito que o @claudiosanches não tenha implementado este recurso porque especificamente o PagSeguro não tem esse recurso de pagamento em dois cartões disponíveis na plataforma em questão (não que eu saiba). O plugin do PagSeguro feito por ele é apenas uma integração, então a plataforma tem que fornecer o recurso para que o desenvolvedor possa implementar isso para ser usado no WooCommerce.

Sigo a procura dessa funcionalidade de pagamento em 2 cartões como você também. Ela é bastante utilizada nos meios físicos e como você falou na plataforma Mercado Livre. Se alguém tiver a informação se outros métodos de pagamento, como o PayPal por exemplo, também tiver essa funcionalidade, por favor, informe aqui.

**Comment by thadeu220** [comment]
sim, claudio sanchez é o envolvido com a mercado pago, para fazer a
integração woocommerce, tanto que o grupo de mercado pago developers ele tá
lá.
então, acredito que a integração woocommerce mercado pago, também deveria
ter o dois cartões transparente.
já tem por redirecionamento, mas, não transparente, e é mais legal no
transparente.

Em qui., 23 de jan. de 2020 às 13:40, lwxbr <notifications@github.com>
escreveu:

**Comment by lwxbr** [comment]
O Plugin do PayPal não apresenta a funcionalidade:
https://wordpress.org/support/topic/e-possivel-fazer-uma-compra-e-pagar-com-multiplos-cartoes/
